### ISSUE TITLE
ACC-264.1, (Level A)_labels _ Not announced

### ISSUE THREAD

**Issue by Priya1Pavan** [kyma-project/busola#3278]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	Labels
Affected Check Point:	ACC-264.1
Short text:	ACC-264.1, (Level A)_labels _ Not announced

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. 
•	Select +Upload YAML.
•	Copy the following code and paste it into the editor: 

apiVersion: v1
kind: Namespace
metadata:
  name: nginx-ns
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: nginx-deployment
  namespace: nginx-ns
  labels:
    app: nginx
spec:
  replicas: 2
  selector:
    matchLabels:
      app: nginx
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - name: nginx
        image: nginx:1.14.2
        ports:
        - containerPort: 80
•	Select Upload
•	Select Close
•	Explore the Monitoring and Health section of the Cluster Details page.
•	In the Events section, select All Messages.
•	Select the sorting icon.
•	In the Sort By section, select Last seen.
•	Select OK.

3. Issue Type: Manual
Observed Behavior:
Labels are not being announced on accessing the ‘Validate resources’ ,’ Namespace nginx-ns’ and ‘Deployment nginx-deployment’ within the dialog ‘Upload Yaml’ 
Screenshot:
 
Expected Behavior:
The corresponding label and group label should be read out, when each & every UI is accessed.

Relationships and information conveyed by the presentation (such as labels) should be identified by assistive technologies so user can connect the accessed UI with associated label.
[ACC-264.1, (Level A)_labels _ Not announced.docx](https://github.com/user-attachments/files/16727283/ACC-264.1.Level.A._labels._.Not.announced.docx)
[Issue processing-09.2021 (2).docx](https://github.com/user-attachments/files/16727284/Issue.processing-09.2021.2.docx)
